src/esn.calendar.libs/app/components/attendee/attendee-item/attendee-item.pug

```diff
@@ -1,19 +1,23 @@
 .attendee-item
   cal-attendee-avatar(attendee='ctrl.attendee')
   .attendee-item-meta
-    .name
+    .name(ng-if="!!ctrl.attendee.id")
       span(
         profile-popover-card='{ source: "ctrl.attendee", property: "id" }'
         profile-popover-card-hide-on-element-scroll='.modal-body'
         profile-popover-card-alternative-title='{{ ::ctrl.attendee.displayName }}'
         profile-popover-card-show-mobile
       )
-        | {{ctrl.attendee.displayName}}
+        | {{ctrl.attendee.displayName}} 
```

Comment:
Can't displayName be defined even if attendee is not an user ?

Reply:
No ,only openpass users contain display Name , external user has only email